docs/shared-content-source/docs/modules/ROOT/pages/app-development-process.adoc

```diff
@@ -27,7 +27,9 @@ The Streamlet model offers an abstraction to:
 
 In the current version, Cloudflow includes backend `Streamlet` implementations for Akka, Apache Spark - Structured Streaming, and Apache Flink. 
 Using these implementations you can write business logic in the native API of the backend.
-Additionally, Cloudflow can be extended with new streaming backends.
+Native support for Flink and Spark streamlets are supported as _legacy_ versions and will be discontinued in future. The current version introduces Flink and Spark integrations with more controls in the hands of the users. Users now have more control on deployment and management of Flink and Spark streamlets while still using the same Cloudflow streamlet API for developing their business logic. However, Akka will continue to be supported natively as in earlier versions.
```

Comment:
I think that we should not yet announce the current integration as "legacy"

Reply:
I agree. Also `native support for`, and `flink native`, seems to have been mixed. We're not going to discontinue the native integration (through cli native kubernetes features). We have to come up with a better name for the current integration style

Reply:
Maybe `supported by the cloudflow operator` vs via the CLI?

Reply:
ok, I will make it sound like both the strategies are available.